srcpkgs/sunshine/template

```diff
@@ -0,0 +1,82 @@
+ https://github.com/LizardByte/tray/archive/4d8b798cafdd11285af9409c16b5f792968e0045.tar.gz>tray.tar.gz
+ https://gitlab.freedesktop.org/wayland/wayland-protocols/-/archive/1.35/wayland-protocols-1.35.tar.gz>wayland-protocols.tar.gz
+ https://gitlab.freedesktop.org/wlroots/wlr-protocols/-/archive/2b8d43325b7012cc3f9b55c08d26e50e42beac7d/wlr-protocols-2b8d43325b7012cc3f9b55c08d26e50e42beac7d.tar.gz>wlr-protocols.tar.gz
+ https://gitlab.com/eidheim/Simple-Web-Server/-/archive/27b41f5ee154cca0fce4fe2955dd886d04e3a4ed/Simple-Web-Server-27b41f5ee154cca0fce4fe2955dd886d04e3a4ed.tar.gz>Simple-Web-Server.tar.gz
+ https://github.com/cgutman/enet/archive/04e27590670a87a7cd40f5a05cda97467e4e25a3.zip>enet.tar.gz"
+checksum="f811b07950a6bfd041e80cc424f95e29d4ba84a8bcdf2a61d5cdc751152c9927
+ 3741283950fb001de383478438fea8dd644bbf043cded1dfc70a4bd87b2bc964
+ 27907cba8c017de220b66e036ed04a39f5eacb9182846700047420aef2f5cd0e
+ c6d9343c8a6306e1b6025bf30893f555b1f1f1a3ffe75ddd3e2e67911adc7339
+ 8ad598c73ad796e0d8280b082cebd82a630d73e73cd3c70057938a6501bba5d7
+ 8390c3fc51796d228577c6887bfa7c3afda42cc18e5967187951c3e0ad343458
+ fee62ecffef3b456e789852787cf534f4712e1a7973bcdd5763baad4b7d2e879
+ dc28f764fbc72f6ac2a96fcace252e6ecc84b308b8d4400820203cc901bae596
+ 689c124192ea1e10dbc907d9d51bd0a30e4bbc1ed3a98a46db601ff8b9f84276
+ 298e90d80c146b07fd71040d08e5184b0baca362258bf26033d7c9b80cdac11f
+ 6e62dfa92ce82487d107b76064cfe2d7ca107c87c239ea9036a763d79c09105a
+ 5bfcca71ba8548bb5d7fdb6ba0ec2c883bf79deca74282c4aba75978837ac000
+ 6179016c556c630e2c1459b38f93db881955e191840e069dd4a98dd3dd1dc56e
+ bc2f1e6975aa5394683c1f82a53459cc45c3cd7a0f9c37b8afc8d77f8818fd58
+ "
+skip_extraction="
+ Simple-Web-Server.tar.gz
+ TPCircularBuffer.tar.gz
+ ViGEmClient.tar.gz
+ build-deps.tar.gz
+ googletest.tar.gz
+ moonlight-common-c.tar.gz
+ nanors.tar.gz
+ nv-codec-headers.tar.gz
+ nvapi-open-source-sdk.tar.gz
+ tray.tar.gz
+ wayland-protocols.tar.gz
+ wlr-protocols.tar.gz
+ enet.tar.gz"
+post_extract() {
+	vsrcextract -C third-party/Simple-Web-Server Simple-Web-Server.tar.gz
+	vsrcextract -C third-party/TPCircularBuffer TPCircularBuffer.tar.gz
+	vsrcextract -C third-party/ViGEmClient ViGEmClient.tar.gz
+	vsrcextract -C third-party/build-deps build-deps.tar.gz
+	vsrcextract -C third-party/googletest googletest.tar.gz
+	vsrcextract -C third-party/moonlight-common-c moonlight-common-c.tar.gz
+	vsrcextract -C third-party/moonlight-common-c/enet enet.tar.gz
+	vsrcextract -C third-party/nanors nanors.tar.gz
+	vsrcextract -C third-party/nv-codec-headers nv-codec-headers.tar.gz
+	vsrcextract -C third-party/nvapi-open-source-sdk nvapi-open-source-sdk.tar.gz
+	vsrcextract -C third-party/tray tray.tar.gz
+	vsrcextract -C third-party/wayland-protocols wayland-protocols.tar.gz
+	vsrcextract -C third-party/wlr-protocols wlr-protocols.tar.gz
+}
+pre_configure() {
+	export BRANCH=master
+	export BUILD_VERSION=${version}
+	export COMMIT=8b21db6
```

Comment:
it would also be good to not hard-code this commit. perhaps set it to `v${version}` (like the git tag is called)